src/persona/persona.resolver.ts

```diff
@@ -0,0 +1,42 @@
+import { Args, Mutation, Query, Resolver } from '@nestjs/graphql';
+import { UseGuards } from '@nestjs/common';
+import { GqlSessionGuard } from '../guards';
+import { ConnectPersonaInput } from '../shared';
+import { PersonaService } from './persona.service';
+import { PersonaType } from './dto/persona.dto';
+import { PersonaInput } from './input';
+import { UpdatePersonaInput } from '../shared/input/update-persona.input';
+
+@Resolver('Persona')
+export class PersonaResolver {
+  constructor(private readonly personaService: PersonaService) {}
+
+  @Query(() => [PersonaType])
+  async getPersona(
+    @Args('condition') input: PersonaInput,
+  ): Promise<PersonaType[]> {
```

Comment:
does 'getPersona' return array?

Reply:
yes - always - for simplified

Reply:
I think it can be confusing when on frontend site we will ask for specified persona and we will always get array with only one element. What do you think about the consuming part of it?

Reply:
Ok - I changed it with the usage of findOne from mongoose. This triggers changes in other code locations. Please check it once more.